site/content/en/docs/Guides/gameserver-lifecycle.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "GameServer Creation, Allocation and Shutdown Lifecycle"
+linkTitle: "GameServer Lifecycle"
+date: 2019-02-01T02:15:18Z
+weight: 15
+description: >
+  Shows the life cycle of `GameServer` creation and integration with the SDK, 
+  when being started and match made.
+---
+
+![Lifecyle Sequence Diagram](http://www.plantuml.com/plantuml/png/XLN1Rjim3BtxAuYSMg3jhaFG5CWwT0kAOa63tdnH92OMwb2U9BVDtu-KxLX9MkmIsARu-FZug3JAHwjjeoZ2augwsgbNz4MJVPuyjclq5370VBcOW0hm-3tmuz6f-TOHXkF4Nsg7i4B_XhwafNSQGqZvFomflpywlFhvU9uyv3vZSAtNc9B7GB5s7qTLOEdCf038WVfQnn-b6f811YYAsIp3X5NaSp2R5KNN4bpVPU5fVbHxL14DnCgxTbkLfE3MtZtKYF7scxq3xMXZjwrNqJegb8xsZS_d3MHWNAQZEksrxE43A6aBtcsik1AMDC1ovRptUqTHMMBKGwM-zyExjDrzatByDfMVcz-ewbXTN1Ra-92tsogoNjpA44ZTM672jQEbrhL15BMqPKegWjEwzG5QCbmUX2PaB5aV39MhpKqINQhadTzUNtTSdfY-j12WAxo745NaKfzmVaPbzgUKxrLT2-y-c6Y_LqX2YCEvMbhnL_OwDBNQ9z9Cc53BgBXHfYdzhck4w3YuHc1ifsNU_-dYfJ6YTNG79ea7-ueJn6pZde77los6A3fcK6A-x12ttItS5EwJ9QO7cyqxJinPQdHYrBWHBSOGgEXsLZF_FJJUlLc3CZsLbznuj-ClZNCrXzm6yCEUyfAaYvVG9g1Zgr-EadHKXNHduoai-ExWvMrY-89tIRqEnQJBcPmktOCb8_PBSWhA61I0gv82ChoDlUppOLQPz3qncTF0p6DiFOMpZ98oQJngvqq-487eqRuQXRYdGqTJKjhTCzPWaDrUXv8iwRerelPYcRpNE9zQwJt9ZpVPS1SRbleJFF7q6ej-nkRgQsx5mAsxLbIIY1UD1yyvBtIanZXRSUyoxKkGaTXGTJibXXrtD1-ShEggQgDnxpHiQ-hX3sc4BP-K_HErjDirDKQy0d5gmJJ6lCIQvJ0e9z8sdtVl-p-99E1QwLVHas_ER5tP-D6x_N0l3gzoBGxF8-W_)
```

Comment:
can we add Makefile step that regenerates this automatically (somehow)?

Reply:
So yep - we totally can do this - plantuml has a command line tool we could install (like dot), and make part of the Makefile :+1: 

I thought this might be easier (one less tool),  but in this case, it's one more website to learn.

I'll add the Makefile step, and make the png local.

Reply:
Done